rest-api/dao/participant_dao.py

```diff
@@ -176,6 +242,9 @@ def get_biobank_ids_sample(self, session, percentage, batch_size):
   def to_client_json(self, model):
     client_json = {
         'participantId': to_client_participant_id(model.participantId),
+        'hpoId': model.hpoId,
```

Comment:
So, the doc here:

https://docs.google.com/document/d/1l7aJqmUYJ3Or6r3lndtfNQ1grfm2SDT2qFKLu8S3ucA/edit#heading=h.34rqknadf1cz

suggests that the API fields be called "awardee", "organization", and "site". "hpoId" already exists, and we should populate it too (with the same value as "awardee" ) for backwards compatibility, but we might deprecate it.

Let me know if those field names sound appropriate to you.